src/librustc_const_eval/eval.rs

```diff
@@ -68,7 +68,7 @@ pub fn lookup_const_by_id<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
             _ => Some((def_id, substs))
         }
     } else {
-        match tcx.sess.cstore.describe_def(def_id) {
+        match ty::queries::describe_def::get(tcx, DUMMY_SP, def_id) {
```

Comment:
@achernyak Hmm, any chance you can squash all of those commits into one *and* rebase on top of master? At this point I believe your branch doesn't conflict in the diff, but #41504 changed how queries are involved (i.e. it'd be just `tcx.describe_def(def_id)` now, here and in the two other places).